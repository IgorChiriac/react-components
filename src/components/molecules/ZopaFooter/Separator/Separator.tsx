import styled from 'styled-components';

import * as colors from '../../../../constants/colors';

const Separator = styled.div`
  background-color: ${colors.neutral.neutral800};
  height: 1px;
`;

export default Separator;
