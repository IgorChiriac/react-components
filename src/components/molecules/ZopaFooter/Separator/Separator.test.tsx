import React from 'react';
import { render } from 'react-testing-library';
import Separator from './Separator';

describe('<Separator />', () => {
  it('renders the component', async () => {
    const { container } = render(<Separator />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
