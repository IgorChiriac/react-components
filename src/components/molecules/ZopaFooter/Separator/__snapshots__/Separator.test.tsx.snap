// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Separator /> renders the component 1`] = `
.c0 {
  background-color: #282F52;
  height: 1px;
}

<div
  class="c0"
/>
`;
